app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cash = {}
```

Comment:
```suggestion
    cash = {}
```
it means 'готівка' :)

Reply:
please choose another name